imap_processing/cdf/tests/test_cdf_attribute_manager.py

```diff
@@ -1,47 +1,121 @@
 from pathlib import Path
 
+# import pytest
+
 from imap_processing.cdf.cdf_attribute_manager import CdfAttributeManager
 
 
+def test_default_attr_schema():
+    """
+    Test function that covers:
+        _load_default_global_attr_schema
+        _load_default_variable_attr_schema
+    """
+
+    # Test
+
+    # Initialize CdfAttributeManager object which loads in default schema
+    cdf_manager = CdfAttributeManager(Path(__file__).parent.parent / "config")
+
+    # Default global tests
+    # Check false case
+    assert cdf_manager.global_attribute_schema["DOI"]["required"] is False
+
+    # Check true case
+    assert cdf_manager.global_attribute_schema["Data_level"]["required"] is True
+
+    # Default variable tests
+    # Check false case
+    assert cdf_manager.variable_attribute_schema['attribute_key']["ABSOLUTE_ERROR"]["required"] is False
+
+    # Check true case
+    assert cdf_manager.variable_attribute_schema['attribute_key']["RESOLUTION"]["required"] is True
+
+
 # @pytest.mark.xfail(reason="Missing IMAP specific global schema")
+
+
 def test_global_attribute():
+    """
+    Test function that covers:
+        load_global_attributes
+        get_global_attributes
+    """
+
+    # Initialize CdfAttributeManager object which loads in default global variables and schema
     cdf_manager = CdfAttributeManager(Path(__file__).parent.parent / "config")
+    # Test that default information was loaded in from "imap_default_global_cdf_attrs.yaml"
```

Comment:
hm, this might actually be a bug within `CdfAttributeManger`. I don't think it should load in any information to start. This is irrelevant to your PR, just a mental note for me to go back and fix that

Reply:
Ah! Okay, I see. This is the code that shows information is loaded in: 
`# Load Default IMAP Global Attributes
        self.global_attributes = CdfAttributeManager._load_yaml_data(
            self.source_dir / DEFAULT_GLOBAL_CDF_ATTRS_FILE
        )
        self.variable_attributes = dict()`

That is line 78 in cdf_attribute_manager.py, and DEFAULT_GLOBAL_CDF_ATTRS_FILE is defined as

`DEFAULT_GLOBAL_CDF_ATTRS_FILE = "imap_default_global_cdf_attrs.yaml"`

in line 12.

Reply:
Hmm ok in that case, leave this test in, but I probably will want to update that lol

Reply:
Should not block or really affect you